dotnet/xml/ns-Microsoft.ML.Transforms.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.ML.Transforms">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>The namespace containing the various <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/transforms">data transformations for text and numerical datasets.</a>.
```

Comment:
Why link to the docs resource when this namespace page will list all its members?

Reply:
to be more specific, linked to the definition of the transforms. 

---
In reply to: [262684552](https://github.com/dotnet/ml-api-docs/pull/76#discussion_r262684552) [](ancestors = 262684552)